scopes/compositions/ui/composition-compare/composition-dropdown.module.scss

```diff
@@ -21,5 +21,4 @@
   max-width: 100%;
   width: 100%;
   padding: 0px;
-  z-index: $nav-z-index + 1;
```

Comment:
this dropdown does not conflict with any other z-index at the moment (so better to leave without)

![Screen Shot 2022-06-14 at 11 58 45](https://user-images.githubusercontent.com/5400361/173548394-d70dd08f-052a-45c5-8394-c9f5eeebc25b.png)